src/reducers/sources.js

```diff
@@ -22,7 +21,8 @@ export type SourcesState = {
   selectedLocation?: {
     sourceId: string,
     line?: number,
-    column?: number
+    column?: number,
+    url: string
```

Comment:
Hi @jasonLaster ,

I'm using selectedLocation's url in `getNewSelectedSourceId()`. I think saving the selectedLocation's url may be the easiest way to obtain the selected file's url.